src/api/routes.ts

```diff
@@ -46,8 +46,10 @@ export const buildPostItemChatMessageRoute = (id: UUID) =>
   `${ITEMS_ROUTE}/${id}/chat`;
 
 export const buildGetMemberBy = (email: string) =>
-  `${MEMBERS_ROUTE}?email=${email}`;
+  `${MEMBERS_ROUTE}/search?email=${email}`;
```

Comment:
Does this route exist? Did we add `search` in the route in `graasp`?

Reply:
this comes from https://github.com/graasp/graasp/pull/76